src/main/java/com/sprint/mission/entity/main/Channel.java

```diff
@@ -0,0 +1,60 @@
+package com.sprint.mission.entity.main;
+
+import com.sprint.mission.entity.addOn.ReadStatus;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.AssertTrue;
+import lombok.*;
+
+import java.io.Serializable;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+import static jakarta.persistence.CascadeType.*;
+
+@Entity
+@EqualsAndHashCode(of = {"channelType", "name"}, callSuper = true)
+@NoArgsConstructor(access = lombok.AccessLevel.PROTECTED)
+@ToString(of = {"channelType", "name", "description"})
+@Getter
+@Schema(description = "채널")
+@Table(name = "channels")
+public class Channel extends BaseUpdatableEntity{
+
+    private ChannelType channelType;
```

Comment:
요한님이 적어주신 DB 스키마와 비교해보면 아래 애노테이션이 추가되어야할 것 같아요.  
```suggestion
    @Enumerated(EnumType.STRING)
    @Column(name = "type", nullable = false)
    private ChannelType channelType;
```